### PR TITLE
Hyperlinks added with comments and translated till frameRate() references 

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
hi.json file earlier only had paragraphs without any hyperlinks now I have added hyperlinks to the text with comments and translated the text is up to date with the current version of p5.js till frameRate references. 

An outcome of Google Summer of Code 2021
Author- Sanjay Singh Rajpoot @SanjaySinghRajpoot
Mentor - Aditya Rana @ranaaditya

### CODE REVIEW

File: src/data/reference/hi.json [hunk 1]
```diff
@@ -606,53 +627,87 @@
...
+              ],
             "returns": "p5 ऑब्जेक्ट"
         },
         "pop": {
-            "description": "पुश () फ़ंक्शन वर्तमान ड्राइंग शैली सेटिंग को रिकॉर्ड करता है, और पॉप () इस सेटिंग को पुनर्स्थापित करता है। ध्यान दें कि ये फ़ंक्शन हमेशा एक साथ उपयोग किए जाते हैं। वे आपको शैली सेटिंग्स और परिवर्तनों को बदलने की अनुमति देते हैं और फिर जो आपके पास था, उसे वापस करते हैं। जब एक नया राज्य पुश () के साथ शुरू किया जाता है, तो यह वर्तमान शैली और परिवर्तन जानकारी पर बनाता है। अधिक नियंत्रण प्रदान करने के लिए पुश () और पॉप () फ़ंक्शन को एम्बेड किया जा सकता है (प्रदर्शन के लिए दूसरा उदाहरण देखें)। पुश () वर्तमान स्थिति और परिवर्तन राज्य सेटिंग्स से संबंधित जानकारी संग्रहीत करता है, जो निम्न कार्यों द्वारा नियंत्रित होता है: भरण (), स्ट्रोक (), टिंट (), स्ट्रोक वाइट (), स्ट्रोककैप (), स्ट्रोकजोन (), इमेजमोड () ), rectMode (), ellipseMode (), colorMode (), textAlign (), textFont (), textMode (), textSize (), textLeading ()।"
```
Comment (ranaaditya) [inline src/data/reference/hi.json:637]: **Please Don't translate name of functions, variables, code snippet, etc.**

File: src/data/reference/hi.json [hunk 2]
```diff
@@ -460,26 +463,37 @@
...
+            "description": [
+                "नकारात्मक बनाने के लिए <a href=\"#/p5/beginContour\">beginContour()</a> और <a href=\"#/p5/endContour\">endContour()</a> फ़ंक्शन का उपयोग करें अक्षर 'O' के केंद्र जैसे आकार के भीतर आकार। <a href=\"#/p5/beginContour\">beginContour()</a> आकृति के लिए शीर्षों को रिकॉर्ड करना शुरू करता है और <a href=\"#/ p5/endContour\">endContour()</a> रिकॉर्डिंग बंद कर देता है। ऋणात्मक आकार को परिभाषित करने वाले शीर्षों को बाहरी आकार से विपरीत दिशा में \"wind\" होना चाहिए। पहले बाहरी दक्षिणावर्त क्रम के लिए कोने बनाएं, फिर आंतरिक के लिए आकृतियाँ, वामावर्त में शीर्ष आकृतियाँ बनाएँ।",
```
Comment (ranaaditya) [inline src/data/reference/hi.json]: use something better alternative for (Negative ) **नकारात्मक** !

File: src/data/reference/hi.json [hunk 3]
```diff
@@ -661,7 +716,11 @@
...
+            "description": [
+                "प्रत्येक सेकंड में प्रदर्शित होने वाले फ़्रेमों की संख्या निर्दिष्ट करता है। उदाहरण के लिए, फ़ंक्शन कॉल फ़्रेमरेट (30) एक सेकंड में 30 बार ताज़ा करने का प्रयास करेगा। यदि प्रोसेसर निर्दिष्ट दर को बनाए रखने के लिए पर्याप्त तेज़ नहीं है, तो फ़्रेम दर नहीं होगी हासिल किया। फ्रेम दर को <a href=\"#/p5/setup\">setup()</a> के भीतर सेट करने की अनुशंसा की जाती है। डिफ़ॉल्ट फ्रेम दर प्रदर्शन की फ्रेम दर पर आधारित होती है (यहां इसे \" भी कहा जाता है) ताज़ा दर\"), जो अधिकांश कंप्यूटरों पर 60 फ़्रेम प्रति सेकंड पर सेट है। 24 फ़्रेम प्रति सेकंड (फ़िल्मों के लिए सामान्य) या इससे अधिक की फ़्रेम दर सुचारू एनिमेशन के लिए पर्याप्त होगी। यह setFrameRate(val) के समान है। ",
```
Comment (ranaaditya) [inline src/data/reference/hi.json]: **Dont covert / Translate Function names, variable name or anything related to code or referrences specific to Processing / Documentation of P5.js**

**Review by ranaaditya — CHANGES_REQUESTED** [review]

File: src/data/reference/hi.json (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json:637]: Yes, I am also trying to do the same.

File: src/data/reference/hi.json (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json]: ऋण (minus) would be a better alternative a indicates minus value

File: src/data/reference/hi.json (hunk 3 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json]: ok

**Review by SanjaySinghRajpoot — COMMENTED** [review]
word replaced with correct phrase

**Pull request merged**